test/built-ins/Temporal/PlainDateTime/prototype/since/roundingmode-halfexpand-default-changes.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.since
+description: Largest unit may be used if enough days are around
```

Comment:
I don't quite get this description?

Reply:
It's a bit awkward, I admit. I've tried to improve it (basically just take the error message and put it into the test metadata).